cmd/controllerinit/run.go

```diff
@@ -89,3 +79,16 @@ func runControllerCmd(cmdName string, args []string) error {
 
 	return cmd.Run()
 }
+
+func configureSystem() error {
+	globalConfigurer, err := global.NewGlobalConfigurer()
+	if err != nil {
+		return fmt.Errorf("creating configurer: %v", err)
```

Comment:
nit: errors should start with capital letter e.g. `Creating configurer`. also let's use `%s` instead of `%v`.